_sass/base/blocks/_school.scss

```diff
@@ -151,6 +151,14 @@
         display: block;
         font-weight: $weight-bold;
         margin-top: -$height;
+
+        &:empty {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 4